### ISSUE TITLE
Consider using System.Text.Json based serializer in rzc

### ISSUE THREAD

**Issue by pranavkm** [aspnet/AspNetCore#7257]
This is not part of the runtime, but it would be one fewer binary that we would need to ship in the SDK.

**Comment by pranavkm** [comment]
This is currently blocked on shipping System.Text.Json in the SDK \ Visual Studio. We can file a new issue once this happens.
